src/scripts/exports.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs');
+const path = require('path');
+
+const currentDir = process.cwd();
+const args = Array.isArray(process.argv) ? process.argv.slice(2) : [];
+const configFile = (args[0] !== 'clean' && args[0]) || 'package.json';
+const config = JSON.parse(fs.readFileSync(path.join(currentDir, configFile)).toString());
+
+const clean = () =>
+	Object.keys(config.exports).forEach(alias => {
+		const aliasFile = path.join(currentDir, alias);
+		fs.existsSync(aliasFile) && fs.unlinkSync(aliasFile);
+	});
+
+const linkExports = () => {
+	Object.keys(config.exports).forEach(alias => {
+		const aliasFile = path.join(currentDir, alias);
+
+		try {
+			require.resolve(config.exports[alias], { paths: [currentDir] });
+			fs.writeFileSync(
+				path.join(aliasFile),
+				`module.exports = require('${path.join(currentDir, config.exports[alias])}');`
```

Comment:
I think we need to use `path.relative`, as using `process.cwd()` puts a full absolute path in the output.